src/System.CodeDom/tests/System/CodeDom/TempFileCollectionTests.cs

```diff
@@ -32,6 +32,8 @@ protected override void AddToCollection(ICollection collection, int numberOfItem
         protected override Type ICollection_NonGeneric_CopyTo_ArrayOfIncorrectValueType_ThrowType => typeof(InvalidCastException);
         protected override Type ICollection_NonGeneric_CopyTo_ArrayOfIncorrectReferenceType_ThrowType => typeof(InvalidCastException);
 
+        [Theory]
+        [MemberData(nameof(ValidCollectionSizes))]
```

Comment:
This seems wrong to me.  The base test has such attributes on it, doesn't it?  I'd expect those to be picked up by xunit, and if they're not, that seems like an xunit bug.  And an xunit analyzer bug that it's flagging this.

Reply:
Yes, this is an xunit analyzer bug. They are picked up correctly. I'll file an issue an meanwhile disable the warning locally here.

Reply:
https://github.com/xunit/xunit/issues/1973